src/scss/components/local-footer/_local-footer.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+.su-local-footer  {
+  .su-local-footer__social-google {
```

Comment:
Where is this being applied?

Reply:
This should be applied when you add a google scholar link (e.g. https://scholar.google.com/citations?user=NztE46IAAAAJ&hl=en&oi=sra) to the `Social Links` section at `/admin/config/system/local-footer`. There was an issue with the dist which I fixed, so it should be displaying properly now.